content/blog/writing-crossguard-policies/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Writing CrossGuard Policies"
+date: 2019-12-19
+meta_desc: "A look at the different types of policies that can be written for Pulumi CrossGuard."
+meta_image: crossguard.png
+authors:
+    - justin-vanpatten
+tags:
+    - crossguard
+    - policy-as-code
+---
+
+We recently announced [a new policy as code solution, CrossGuard]({{< relref "/blog/announcing-crossguard-preview" >}}) that validates policies at deployment time. Policies are expressed as code and are used to prevent the creation of out-of-compliance resources. Since policies are written using full-blown programming languages, it's possible to do interesting things such as [combining IAM Access Analyzer and Pulumi CrossGuard]({{< relref "/blog/aws-iam-access-analyzer-and-crossguard" >}}). In this post, we'll take a closer look at the different types of policies that can be written.
+
+<!--more-->
+
+## Authoring Policies
+
+A policy contains specific logic you want to enforce. For example, you may want to prevent the creation of public, world-readable storage objects (e.g. on AWS S3, Azure BlobStore, etc.) or prevent the creation of a virtual machine without the proper security groups in-place. Policies are run during `pulumi preview` and `pulumi update`, ensuring that cloud resource definitions are in compliance.
```

Comment:
My understanding is that "security groups" is an AWS-specific thing. Perhaps use "proper firewalls" or "secure networking rules" instead, if that is more common?